ui/src/scenes/Schema/Table/ContextualMenu/services.ts

```diff
@@ -33,7 +47,13 @@ export const formatTableSchemaQueryResult = (
       }
     }
 
-    return `${trim(query)})`
+    query += ")"
+
+    if (designatedName) {
+      query += ` timestamp ${designatedName}`
```

Comment:
The designated timestamp part at the end needs parens, so it should be

```
      query += ` timestamp (${designatedName})`
```

Reply:
oh yes my bad good catch, will resolve that today